stubs/model-factory.stub

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace {{ namespace }};
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+use {{ namespace }}\Contracts\{{model_factory_interface}}
```

Comment:
Is that naming convention using snake? It would be greate using camelCase since we using DTO

Reply:
dto toArray() generate it into snake case, since dto to array resolv database column naming